scripts/west_commands/completion.py

```diff
@@ -30,10 +57,9 @@ def do_add_parser(self, parser_adder):
 
         # Remember to update west-completion.bash if you add or remove
         # flags
-        parser.add_argument('shell', nargs=1, choices=['bash'],
-                            help='''Select the shell that which the completion
-                            script is intended for.
-                            Currently only bash is supported.''')
+        parser.add_argument('shell', nargs=1, choices=['bash', 'zsh'],
+                            help='''Shell that which the completion
+                            script is intended for.''')
```

Comment:
It would be awesome if we could auto-detect the shell if called without a shell type argument. Not a blocker, of course.

Reply:
For the future :)

Reply:
`shell = os.getenv("SHELL").split("/")[-1]`

Reply:
Thanks!